src/main/kotlin/racingcar/RacingCarManager.kt

```diff
@@ -0,0 +1,44 @@
+package racingcar
+
+import racingcar.domain.Car
+import racingcar.domain.DriveConditionImpl
+import racingcar.presentation.InputManager
+import racingcar.presentation.OutPutManager
+
+class RacingCarManager(
+    val inputManager: InputManager,
+    val outPutManager: OutPutManager
+) {
+    fun startGame() {
+        val numberOfCar = inputManager.inputNumberOfCar()
+        val numberOfCount = inputManager.inputNumberOfCount()
+        val carList: MutableList<Car> = mutableListOf()
```

Comment:
```suggestion
        val cars: MutableList<Car> = mutableListOf()
```

carList보다는 cars가 더 어울리지 않을까요?